build_tools/cmake_configure.sh

```diff
@@ -82,7 +82,7 @@ echo "Using llvm-lit: $LLVM_LIT"
 # Write a .env file for python tooling.
 function write_env_file() {
   echo "Updating $build_dir/.env file"
-  echo "PYTHONPATH=\"$(realpath "$build_dir/python_native"):$(realpath "$build_dir/python")\"" > "$build_dir/.env"
+  echo "PYTHONPATH=\"$(realpath "$build_dir/python_native"):$(realpath "$build_dir/python"):$(realpath "$install_mlir/python")\"" > "$build_dir/.env"
```

Comment:
Question for you Stella: is python_native still relevant here? We don't seem to be creating that directory anymore.

Reply:
No, python_native is gone.

Reply:
Removed it.